sysutils/wxHexEditor/files/0001-Fix-build-for-wxWidgets-v3.1.2.patch

```diff
@@ -0,0 +1,40 @@
+From ebe2449fac22089825d124935a215fd1c0739403 Mon Sep 17 00:00:00 2001
+From: GT <george@tsimperopoulos.com>
+Date: Tue, 10 Sep 2019 23:52:19 +0100
+Subject: [PATCH] Fix build for wxWidgets v3.1.2
+
+---
+ src/HexDialogs.cpp                  | 2 +-
+ src/HexEditorCtrl/HexEditorCtrl.cpp | 6 +++---
+ 2 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git src/HexDialogs.cpp src/HexDialogs.cpp
+index 260571b..9c312ba 100644
+--- src/HexDialogs.cpp
++++ src/HexDialogs.cpp
+@@ -420,7 +420,7 @@ void FindDialog::OnChar( wxKeyEvent& event ){
+ 	}
+ 
+ void FindDialog::EventHandler( wxCommandEvent& event ){
+-	WX_CLEAR_ARRAY(parent->HighlightArray )
++	WX_CLEAR_ARRAY(parent->HighlightArray);
+ 	parent->HighlightArray.Shrink();
+ 
+ 	if( event.GetId() == btnFind->GetId())
+diff --git src/HexEditorCtrl/HexEditorCtrl.cpp src/HexEditorCtrl/HexEditorCtrl.cpp
+index 37a6e4b..6f3a4a0 100644
+--- src/HexEditorCtrl/HexEditorCtrl.cpp
++++ src/HexEditorCtrl/HexEditorCtrl.cpp
+@@ -64,9 +64,9 @@ HexEditorCtrl::~HexEditorCtrl( void ){
+ 	Dynamic_Disconnector();
+ 	Clear();
+ 
+-	WX_CLEAR_ARRAY(MainTagArray)
+-	WX_CLEAR_ARRAY(HighlightArray)
+-   WX_CLEAR_ARRAY(CompareArray)
```

Comment:
Ditto; how did this compile before?

Reply:
This is a patch from upstream to fix the build with wxWidgets 3.2.
“Before” it worked with an earlier version of wxWidgets.